### PR TITLE
improve the Linux configuration instructions

### PR DESCRIPTION
**Pull request by anarcat**
This is a bunch of patches to improve the Linux instructions for the
"red" VPN. Each commit has detailed instructions, but here's an
overview:

 * remove hardcoded IP address
 * add port 53 to sample config
 * cleanup configuration file
 * remove (broken) non-root instructions
 * sync update-resolv-conf script with Debian

### CODE REVIEW

File: pages/vpn/vpn-red/linux/en.text [hunk 1]
```diff
@@ -76,14 +76,10 @@ $ sudo apt install openvpn resolvconf
...
 
-Download [[this script for updating the resolver => https://riseup.net/vpn/vpn-red/update-resolv.conf]] and place it in /etc/openvpn/update-resolv.conf
+If it is not already provided by your distribution (it is in Debian), download [[this script for updating the resolver => https://riseup.net/vpn/vpn-red/update-resolv-conf]] and place it in @/etc/openvpn/update-resolv-conf@
 
 Then you need to download the [[configuration file => https://riseup.net/vpn/vpn-red/riseup.ovpn]] and you need to change it for your credentials. The @auth.txt@ is a file with two lines, the first is your riseup.net login name, the second is a [[VPN Secret->/vpn/vpn-secret]].
 
-In order for openvpn to create the tun device it must be started as user root. However, it will change to the user and group you specify. For @user@ and @group@ options, you can replace with your username or make auth.txt and RiseupCA.pem readable by @nobody@. Alternately, you can leave off those options to run openvpn as root.
-
-For more options, see @man openvpn@.
-
```
Comment (micah) [inline pages/vpn/vpn-red/linux/en.text]: I'm curious why this section is being removed? Is it no longer true, or?

File: pages/vpn/vpn-red/riseup.ovpn [hunk 2]
```diff
@@ -21,6 +20,8 @@ auth-user-pass
 # alternately:
 # auth-user-pass ~/vpn/auth.txt
 
+dev tun
+
```
Comment (micah) [inline pages/vpn/vpn-red/riseup.ovpn]: Is there some purpose to moving this to a different location?

**Comment by micah** [comment]
anarcat <notifications@github.com> writes:

> I see. Well I think that's rather unusual - what about if that's
> available as a comment instead?

I'd rather leave it as the IP address, because people are more
frequently simply copying and pasting the config file, rather than
reading the options and deciding what to do with them. When people copy
and paste a config file, and it doesn't work then they come to ask us
why not. Defaulting to it working seems to me better than making it a
comment that we have to tell people to uncomment in a weird network
environment.

> Otherwise if you feel strongly about this, I can fixup that commit to,
> instead, document why that's an IP address to resolve the WTF I felt
> when reading the config. :)

Besides making it more difficult for us to move a server, why is the IP
such a WTF?

**Comment by anarcat** [comment]
On 2018-09-24 07:53:07, micah wrote:
> anarcat <notifications@github.com> writes:
>
>> I see. Well I think that's rather unusual - what about if that's
>> available as a comment instead?
>
> I'd rather leave it as the IP address, because people are more
> frequently simply copying and pasting the config file, rather than
> reading the options and deciding what to do with them. When people copy
> and paste a config file, and it doesn't work then they come to ask us
> why not. Defaulting to it working seems to me better than making it a
> comment that we have to tell people to uncomment in a weird network
> environment.

Well, then you'll want to switch to TCP and port 443 by default. ;)

>> Otherwise if you feel strongly about this, I can fixup that commit to,
>> instead, document why that's an IP address to resolve the WTF I felt
>> when reading the config. :)
>
> Besides making it more difficult for us to move a server, why is the IP
> such a WTF? 

Because those are difficult to read?

I don't have a strong opinion on this - if you prefer, I'll put it back
the way it was. :)

A.

-- 
Every time I see an adult on a bicycle I no longer despair for the
future of the human race.
                         - H. G. Wells

**Pull request closed without merging**